platform/darwin/src/MGLStyle.mm

```diff
@@ -34,6 +34,7 @@
 
 @interface MGLStyle()
 @property (nonatomic, weak) MGLMapView *mapView;
+@property (readonly, copy, nullable) NSURL *URL;
```

Comment:
Fine for now, but it would be nice to make this public at some point, especially once we have #6386.